### ISSUE TITLE
[BUG]: Aspect ratio on ultrawide screens

### ISSUE THREAD

**Issue by kjljxybr** [Paliverse/DualSenseX#170]
**Describe the bug**
There is locked aspect ratio, and it looks super weird on 32:9 & maybe on 21:9 screens 

**To Reproduce**
1. Buy double WQHD monitor
2. Open the app
3. Try to change window size in single directon (e.g. X or Y only)

**Expected behavior**
I expected to be able to resize the window to any aspect ratio, but i couldn't

**Comment by Paliverse** [comment]
Hmm, i don't have an ultrawide monitor so unable to test, but there is a possibility of having options for window sizes

**Comment by kjljxybr** [comment]
let me show you what i mean

normal window behavior on its resize is [here](https://youtu.be/GG4Rc4m7obo)
DSX window behavior on its resize is [here](https://youtu.be/NMd3umDw3SA)

**Comment by Paliverse** [comment]
Yes I know it’s locked to to the scale that I’m using. I do have DSX calculate the screen width and height and divide each one by 1.3 and then that would determine the width/height. I will be adding options soon for this in settings.
